src/Pact/Types/ExpParser.hs

```diff
@@ -73,13 +73,13 @@ data Cursor = Cursor
 instance Default Cursor where def = Cursor def def
 
 -- | adapted from Text.Megaparsec.Stream
-defaultAdvance1
+_defaultAdvance1
```

Comment:
Realizing we can ditch this now. @fosskers lmk when you're done with nix stuff and I'll remove this